### PR TITLE
all funtions from backend built in frontend

### PR DESCRIPTION
**Pull request by LetterBe**

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-import TodoForm from "./todoForm/TodoForm";
-import TodoList from "./todoList/todoList";
+import TodoPage from "./Todos/TodoPage";
+
 
 function App() {
 
+    /*const todoUpdated = ()=> {
+        alert("succeded!")
+    };*/
```
Comment (Droggelbecher92) [inline frontend/src/App.tsx:9]: A wise man said once: "If you don't need it now, remove it."

**Review by Droggelbecher92 — APPROVED** [review]
Very good! 👍 Left some notes for errorhandling, but nothing critical, so you can look into it with your next pullrequest. Good job :)

File: frontend/src/Todos/TodoEdit.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        fetch(`${process.env.REACT_APP_BASE_URL}/api/todos/${props.todoToChange.id}`, {
+            method: 'DELETE',
+        })
+            .then(response => {
+                if (response.status === 200) {
+                    return response.json()
+                }
+                throw new Error("Error to delete Task")
+            })
+            .then(() => props.onTodoChange());
```
[the remark this answers is not in the document]
Comment (LetterBe) [reply frontend/src/Todos/TodoEdit.tsx:102]: done!

File: frontend/src/Todos/TodoEdit.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+
+    const AddOrEdit = () => {
+        if (props.todoToChange.id != null) {
+            editTodo()
+        } else {
+            addTodo()
+        }
+    }
+    const addTodo = () => {
```
[the remark this answers is not in the document]
Comment (LetterBe) [reply frontend/src/Todos/TodoEdit.tsx:33]: the one in the repository of neuefische doesnt not, write now looking on it . MY errorMessage ist set to ' ', dont understand what you might try to tell me

**Review by T-Kutzner — APPROVED** [review]
Well done ;)

**Pull request merged**